ansible/templates/usb/scripts/prompt-config.sh.j2

```diff
@@ -685,15 +698,16 @@ nicsup()
 		ifconfig $admin_iface_active inet $admin_ip netmask $admin_netmask up
 	fi
 
-	if [[ -n ${external_nic} ]]; then
-		if [ -n "$external_vlan_id" ]; then
-			vlan_opts="-v $external_vlan_id"
+	# create an interface for each nic tag with defined IP address
+	for ntag in "${!nictag_IPs[@]}"; do
+		if [ -n "${nictag_vlans[$ntag]}" ]; then
+			vlan_opts="-v ${nictag_vlans[$ntag]}"
 		fi
-
-		dladm create-vnic -l $external_iface $vlan_opts external0
-		ifconfig external0 plumb
-		ifconfig external0 inet $external_ip netmask $external_netmask up
-	fi
+		dladm create-vnic -l ${nictag_ifaces[$ntag]} $vlan_opts "${ntag}0"
```

Comment:
A test for non-empty values used in this code block could protect us from future mistakes. I mean that this code should only run when: ```[[ -n "${nictag_ifaces[$ntag]}" && "${nictag_IPs[${ntag}]}" ....]```.  What do you think?